incydr/_alerts/client.py

```diff
@@ -74,6 +75,9 @@ def get_details(self, alert_ids: Union[str, List[str]]) -> List[AlertDetails]:
         """
         Get full details for a set of alerts.
 
+        The `query-details` endpoint accepts a maximum of 100 ids per request, if `alert_ids` is > 100, multiple
+        requests will be made and results will be combined into a single list.
```

Comment:
this is cool 😎  also feel like this is probably worth some sort of unit test

Reply:
test added